documentation/modules/exploit/linux/http/denyall_waf_exec.md

```diff
@@ -0,0 +1,37 @@
+## Vulnerable Application
+
+This module exploits the command injection vulnerability of DenyAll Web Application Firewall. Unauthenticated user can execute a terminal command under the context of the web server user.
+
+**Vulnerable Application Installation Steps**
+
+It's possible to have trial demo for 15 days at Amazon Marketplace.
+[https://aws.amazon.com/marketplace/pp/B01N4Q0INA?qid=1505806897911](https://aws.amazon.com/marketplace/pp/B01N4Q0INA?qid=1505806897911)
+
+You just need to follow instruction above URL.
+
+## Verification Steps
+
+A successful check of the exploit will look like this:
+
+```
+msf > use exploit/linux/http/denyall_exec 
+msf exploit(denyall_exec) > 
+msf exploit(denyall_exec) > set RHOST 35.176.123.128
```

Comment:
These look to be public IPs (unverified), you may want to change them to something else like 1.1.1.1 and 2.2.2.2

Reply:
It was my AWS London instance IP that I've used for testing purpose. It's not alive anymore.